src/hotspot/hotspots.ts

```diff
@@ -369,3 +369,34 @@ export function showHotspotDetails(hotspotDetails: ShowRuleDescriptionParams, ho
 
   hotspotDetailsPanel.reveal();
 }
+
+export async function changeHotspotStatus(hotspotServerKey: string, fileUriAsSting: string,
+                                                  languageClient: SonarLintExtendedLanguageClient) {
+  const fileUri = vscode.Uri.parse(fileUriAsSting);
+  const workspaceFolder = vscode.workspace.getWorkspaceFolder(fileUri);
+  return doChangeHotspotStatus(hotspotServerKey, fileUriAsSting, workspaceFolder, languageClient);
+}
+
+// visible for testing
+export async function doChangeHotspotStatus(hotspotServerKey: string, fileUriAsSting: string,
+                                            workspaceFolder:vscode.WorkspaceFolder,
+                                            languageClient: SonarLintExtendedLanguageClient) {
+  const statusesResponse =
```

Comment:
`allowedHotspotStatuses` maybe?

Reply:
Addressed in [7c93c61](https://github.com/SonarSource/sonarlint-vscode/pull/411/commits/7c93c6130d509717de007ea8ec719bd89279f085)